CONTRIBUTING.md

```diff
@@ -73,7 +73,7 @@ guidelines:
 
  7. For user visible changes (API changes, behaviour changes, ...),
     consider adding a note in [CHANGES.md](CHANGES.md).
-    This could be a summarising description of the change, and could
+    This could be a summarizing description of the change, and could
```

Comment:
AAUI OpenSSL generally prefers UK spelling, which uses `s`, rather than US spelling, which uses `z`. There are a few of these in this PR; just commenting on this the once.